src/deploy-tes-on-azure/AzureEnvironmentExtension.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Microsoft.Azure.Management.ResourceManager.Fluent;
+
+namespace deploytesonazure
+{
+    public static class AzureEnvironmentExtension
+    {
+        public static string GetBlobEndPointSuffix(this AzureEnvironment env)
+        {
+            return ".blob." + env.StorageEndpointSuffix;
+        }
+
+        public static bool IsAvailableEnvironmentName(string envName)
+        {
+            foreach (var azEnv in AzureEnvironment.KnownEnvironments)
+            {
+                if (0 == string.Compare(azEnv.Name, envName, true))
```

Comment:
let's use this convention instead for consistency:

if (string.Equals(azEnv.Name, envName, StringComparison.OrdinalIgnoreCase))

Reply:
Already fixed.
```csharp
        public static bool IsEnvironmentNameAvailable(string envName)
        {
            foreach (var azEnv in AzureEnvironment.KnownEnvironments)
            {
                if (string.Equals(azEnv.Name, envName, StringComparison.OrdinalIgnoreCase))
                {
                    return true;
                }
            }
            return false;
        }
```